### PR TITLE
Update print statements and encoding for python 3

### PR DESCRIPTION
**Pull request by miriam-rittenberg**

### CODE REVIEW

File: attachandrun [hunk 1]
```diff
@@ -13,19 +14,20 @@ def extract(args):
...
 def invoke(args):
     try:
         process = subprocess.Popen(args,
-                                   stdout=subprocess.PIPE)
+                                   stdout=subprocess.PIPE, )
         result, _ = process.communicate()
         if process.returncode != 0:
             raise Exception("Non-zero return code from '%s'" % (args[0],))
-        return result.strip()
+	# We can't pass encoding to Popen because that's not python2 compatible
```
Comment (andersk) [inline attachandrun]: You could pass `universal_newlines`.

Comment (miriam-rittenberg) [reply attachandrun]: Ok, I changed it

**Review by kaduk — APPROVED** [review]
Thanks for the update; `universal_newlines` looks good.

**Comment by kaduk** [comment]
(This one I didn't adjust the commit message for, so we get the pretty purple "merged" button.)
Once more, thanks for putting this together and sorry for the delay in review.

**Pull request merged**